### DISCUSSION TITLE
amd64/ prefix being removed by renovate-bot; amd64 arch not ready for pull

### DISCUSSION THREAD

**Discussion by shawntoffel** [renovatebot/renovate#11463] [Request Help] [answered]
Hello!

I had the same issue as  #10662. The version was being bumped before the amd64 image was available to pull. I added the amd64/ prefix to the image per the advice in the issue. This did indeed work for several months, however renovate-bot has recently began to remove the prefix as a part of the update commit causing my build to fail again.

See an example here: https://github.com/kotatsuclub/docker-ghost/commit/6e0bfa5ca081bea4797072556df5f53395af9824

Just to experiment a bit I specified `amd64/ghost` in matchPackageNames, however that did not seem to work. 
```
  "packageRules": [
    {
      "matchPackageNames": ["amd64/ghost"],
      "updateTypes": ["major"],
      "automerge": false
    }
  ]
```

My goal is to ensure the version bump of the `ghost` docker image in my Dockerfile does not occur before the amd64 ghost image is available to pull. 

Is there a way to achieve this in renovate config or a change to my Dockerfile? Or is this a bug or change in behavior, expected or otherwise?

This PR #10997 seems relevant to my problem and the timing of the merge seems to line up, however I'm unsure if it's the cause. 

Thank you.

**Comment by rarkins** [comment, marked as answer]
If Renovate strips the prefix then please create a minimal reproduction repository and file a bug report issue.

**Reply by shawntoffel** [reply]
Created #11464. Thank you.
